packages/opencensus-core/src/trace/sampler/types.ts

```diff
@@ -14,36 +14,13 @@
  * limitations under the License.
  */
 
-import {Logger} from '../../common/types';
-import {Exporter} from '../../exporters/types';
-import {PluginNames} from '../instrumentation/types';
 
-/** This interface represent the probability of a tracer. */
+/** This interface represent a sampler . */
 export interface Sampler {
   /**
    * A Sampler description name
    */
   description: string;
```

Comment:
Mark this as `readonly` as well. Also, let's expand on the documentation a little bit:
```
A string that uniquely describes the sampling behavior of this instance.
```

Reply:
Done.